integrations-and-supported-tools/arize/notebooks/Neptune_Arize.ipynb

```diff
@@ -0,0 +1,237 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arize integration guide\n",
+    "Arize and Neptune are MLOps tools that aim to improve connected but different parts of your ML pipeline and workflow.\n",
```

Comment:
Arize and Neptune are MLOps tools that aim to improve different (but related) parts of your ML pipeline and workflow.